packages/components/Button/package.json

```diff
@@ -15,7 +15,9 @@
     "just": "fluentui-scripts",
     "clean": "fluentui-scripts clean",
     "lint": "fluentui-scripts eslint",
-    "test": "fluentui-scripts jest",
+    "test": "yarn test-iOS && yarn test-win32",
+    "test-iOS": "fluentui-scripts jest -c __tests__/iOS/jest.config.js",
+    "test-win32": "fluentui-scripts jest -c __tests__/win32/jest.config.js",
```

Comment:
If we follow the same structure everywhere, could we modify the script invoked with `fluentui_scripts` to take a `-- platform` arg that then chooses the right config? Guessing on how to make this a little more ergonomic. If we can make it an argument like `--platform` then we don't need extra `test-<platform>` scripts

Reply:
I think it could also be nice to have a per-platform command so that you don't have to run /all the tests/ in order to iterate on a failure if you need to look into something, so I would say there's value in having per-platform commands

Reply:
Let me rephrase:

`yarn test` would still do some default platform
`yarn test --platform ios` would test iOS specifically, and pass the iOS jest config.

Reply:
Ah okay. Would you be open to that change being a follow up? I think I'll need to add a new script in order to implement it, since the args currently have to be an option that the jest CLI accepts.

Reply:
Yeah that's fine!

Reply:
Leaving things to be more ergonomic as a follow up makes sense! Thanks =)